### PR TITLE
btle: Add definitions for Bluetooth 5.0 control PDUs

### PR DESCRIPTION
**Pull request by rugeGerritsen**
The changes only support using scapy as a packet builder.
Dissection is not yet implemented.

I've used https://github.com/secdev/scapy/pull/2215 by @Matheus-Garbelini  as a starting point.

closes #2215

### CODE REVIEW

**Comment by Matheus-Garbelini** [comment]
@rugeGerritsen thanks a lot. Dissection may be implemented after adding `def dissection_done` to the BTLE_DATA class as suggested by @guedou here: https://github.com/secdev/scapy/issues/2192#issuecomment-520589684
In short, it appears empty PDUs or packets with no attributes cannot be dissected if they are empty. One workaround is to forcibly call the lines bellow: https://github.com/secdev/scapy/blob/962a95ab47ccc2516e631c97c03b8ef807b33f06/scapy/layers/bluetooth4LE.py#L362-L363

I couldn't confirm this at the time so feel free to try this out.
Thanks a lot for following up on this PR.

**Comment by rugeGerritsen** [comment]
@Matheus-Garbelini , thanks for your feedback. I will leave the dissection as is for now. The last push only initializes the base class of `BTLEPhysField` in a more pythonic way. Unfortunately it still fails with:
```
Expression "(super(BTLEPhysField, self).__init__(name, default, -8, ['phy_1m', 'phy_2m', 'phy_coded']), )" is assigned to nothing
``` 
I'm familiar enough with pylint to understand what is wrong here

File: scapy/layers/bluetooth4LE.py [hunk 1]
```diff
@@ -128,6 +129,33 @@ def getfield(self, pkt, s):
...
+                             'le_ext_adv',
+                             'le_periodic_adv',
+                             'ch_sel_alg',
+                             'le_pwr_class'])
+
+
+class BTLEPhysField(FlagsField):
+    def __init__(self, name, default):
+        super(BTLEPhysField, self).__init__(name, default, -8,
+                                            ['phy_1m', 'phy_2m', 'phy_coded']),
```
Comment (gpotter2) [inline scapy/layers/bluetooth4LE.py:156]:
```suggestion
        super(BTLEPhysField, self).__init__(
            name, default, -8,
            ['phy_1m', 'phy_2m', 'phy_coded']
        )
```

File: scapy/layers/bluetooth4LE.py [hunk 2]
```diff
@@ -128,6 +129,33 @@ def getfield(self, pkt, s):
...
+                             'll_privacy',
+                             'ext_scan_filter',
+                             'le_2m_phy',
+                             'tx_mod_idx',
+                             'rx_mod_idx',
+                             'le_coded_phy',
+                             'le_ext_adv',
+                             'le_periodic_adv',
+                             'ch_sel_alg',
+                             'le_pwr_class'])
```
Comment (gpotter2) [inline scapy/layers/bluetooth4LE.py:150]: I feel like the formatting could be made a bit better.

```suggestion
        super(BTLEFeatureField, self).__init__(
            name, default, -64,
            ['le_encryption',
             'conn_par_req_proc',
             'ext_reject_ind',
             'slave_init_feat_exch',
             'le_ping',
             'le_data_len_ext',
             'll_privacy',
             'ext_scan_filter',
             'le_2m_phy',
             'tx_mod_idx',
             'rx_mod_idx',
             'le_coded_phy',
             'le_ext_adv',
             'le_periodic_adv',
             'ch_sel_alg',
             'le_pwr_class']
        )
```

**Review by gpotter2 — COMMENTED** [review]
Apart from two minor cosmetic comments (that you can probably ignore), this LGTM !
Thanks for the PR

**Comment by rugeGerritsen** [comment]
@gpotter2 , thanks for your review!

**Review by gpotter2 — APPROVED** [review]

**Pull request merged**